examples/nifpga/read_write_fifo_example.py

```diff
@@ -0,0 +1,162 @@
+r""" Read and Write to indicators, controls, Arrays and FIFO.
+
+The gRPC API is built from the C API. NI-FPGA documentation is installed with the driver at:
+    C:\Program Files (x86)\National Instruments\FPGA Interface C API\capi.chm
+
+Getting Started:
+
+To run this example, you need to provide the correct file path and bitfile signature.
```

Comment:
Same comment as above on "correct file path..."

Reply:
Updated the comment.